src/components/Form/Form.js

```diff
@@ -1,28 +1,28 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-class Form extends React.Component {
+class Form extends Component {
 
-  submitFn = ( event ) => {
+  handleSubmitForm =  event  => {
```

Comment:
jedną spacje mniej przed event i jedną mniej po bo wyglada to dziwnie sam przyznasz